src/Calculator/Resources/en-US/Resources.resw

```diff
@@ -4106,7 +4106,167 @@
+    <value>Missing x or y variable.</value>
+    <comment>Error that occurs during graping when the equation is missing x or y.</comment>
+  </data>
+  <data name="InvalidEquationSyntax" xml:space="preserve">
+    <value>Invalid equation syntax.</value>
+    <comment>Error that occurs during graping when an invalid syntax is used.</comment>
+    </data>
+    <data name="EmptyExpression" xml:space="preserve">
+    <value>The expression is empty.</value>
+    <comment>Error that occurs during graping when the expression is empty</comment>
+  </data>
+    <data name="EqualWithoutEquation" xml:space="preserve">
+    <value>Equal was used without an equation.</value>
+    <comment>Error that occurs during graping when equal is used without an equation. Ex: sin(x=y)</comment>
+  </data>
+  <data name="ExpectParenthesisAfterFunctionName" xml:space="preserve">
+    <value>Paranthesis missing after function name.</value>
+    <comment>Error that occurs during graping when paranthesis are missing after a function.</comment>
+    </data>
+  <data name="IncorrectNumParameter" xml:space="preserve">
+    <value>A function has the incorrect number of parameters.</value>
+    <comment>Error that occurs during graping when a function has the wrong number of parameters</comment>
+  </data>
+  <data name="InvalidVariableNameFormat" xml:space="preserve">
+    <value>The variable name is invalid.</value>
+    <comment>Error that occurs during graping when a variable name is invalid.</comment>
+  </data>
+  <data name="BracketMismatch" xml:space="preserve">
+    <value>The equation is missing an opening bracket.</value>
+    <comment>Error that occurs during graping when a { is missing</comment>
+  </data>
+  <data name="UnmatchedBracket" xml:space="preserve">
+    <value>The equation is missing a closing bracket.</value>
+    <comment>Error that occurs during graping when a } is missing.</comment>
+  </data>
+  <data name="CannotUseIInReal" xml:space="preserve">
+    <value>"i" and "I" cannot be used as variable names.</value>
+    <comment>Error that occurs during graping when i or I is used.</comment>
+  </data>
+  <data name="GeneralError" xml:space="preserve">
+    <value>The equation could not be graphed.</value>
+    <comment>General error that occurs during graping.</comment>
+  </data>
+  <data name="InvalidNumberDigit" xml:space="preserve">
+    <value>The digit could not be resolved for the given base.</value>
+    <comment>Error that occurs during graping when trying to use bases incorrect. Ex: base(2,1020).</comment>
+  </data>
+  <data name="InvalidNumberBase" xml:space="preserve">
+    <value>The base must be greater than 2 or less than 36.</value>
+    <comment>Error that occurs during graping when the base is out of range.</comment>
+  </data>
+  <data name="InvalidVariableSpecification" xml:space="preserve">
+    <value>A function in the equation requres a variable in a certain argument position.</value>
```

Comment:
I find this error message confusing. Maybe something like "Required variable is missing."